ManatiCR/vitaminau
# VIT-40, VIT-44 and VIT-45, compound fields and textarea counters

# Tasks [VIT-40](https://manati.atlassian.net/browse/VIT-40?atlOrigin=eyJpIjoiODY4ODQ4YjUyMDMzNDk5MWIzMWZkZGM4ZDFlMTM5YWUiLCJwIjoiaiJ9), [VIT-44](https://manati.atlassian.net/browse/VIT-44?atlOrigin=eyJpIjoiODY4ODQ4YjUyMDMzNDk5MWIzMWZkZGM4ZDFlMTM5YWUiLCJwIjoiaiJ9), [VIT-45](https://manati.atlassian.net/browse/VIT-45?atlOrigin=eyJpIjoiODY4ODQ4YjUyMDMzNDk5MWIzMWZkZGM4ZDFlMTM5YWUiLCJwIjoiaiJ9)

Update of the "Uso de instalaciones" form. 
Update of the "Venta de productos" form.
Adding character counter in all's text area. 

This changes was making because helps users to know which count of characters have to write in a textarea, and the custom fields are a specified in excel sheet.

# Installation
- Run the following command
`ahoy drush cr && ahoy drush cim -y`

# Test
- Login as admin user:
  - User: admin
  - Pass: admin
- Go to the following url's: 

1. http://vitaminau.docker/form/webform-entrepreneurship-support
    - [x] When you write in text area fields, a character counter should appear and show how many characters are left.
2. http://vitaminau.docker/form/webform-licensed-technologies
    - [x] When you write in text area fields, a character counter should appear and show how many characters are left.
3. http://vitaminau.docker/form/webform-installations-use
- [VIT-44](https://manati.atlassian.net/browse/VIT-44?atlOrigin=eyJpIjoiODY4ODQ4YjUyMDMzNDk5MWIzMWZkZGM4ZDFlMTM5YWUiLCJwIjoiaiJ9)
    - [x] Once on this page, go to the "Equipo especializado" section.
    - [x] When you press the "+" button, a new row is added.
    - [x] When you press the "-" button, a row is deleted.
    - [x] You should be able to submit the form without errors.
- [x] VIT-40: When you write in text area fields, a character counter should appear and show how many characters are left.

4. http://vitaminau.docker/form/sale-of-products
  - [VIT-45](https://manati.atlassian.net/browse/VIT-45?atlOrigin=eyJpIjoiODY4ODQ4YjUyMDMzNDk5MWIzMWZkZGM4ZDFlMTM5YWUiLCJwIjoiaiJ9)
    - [x] Once on this page, go to the "Presentación de producto" section.
    - [x] When you press the "+" button, a new row is added.
    - [x] When you press the "-" button, a row is deleted.
    - [x] You should be able to submit the form without errors.
- [x] VIT-40: When you write in text area fields, a character counter should appear and show how many characters are left.

5. http://vitaminau.docker/form/work-in-progress
- [x] When you write in the fields "Plan de estudios" and "¿Qué mejoraría?", a character counter should appear and show how many characters are left.
